compilers/concrete-optimizer/concrete-optimizer-cpp/src/cpp/concrete-optimizer.cpp

```diff
@@ -929,6 +929,13 @@ struct operator_new<T, decltype(T::operator new(sizeof(T)))> {
 };
 } // namespace detail
 
+template <typename T>
```

Comment:
?

Reply:
That's automatically generated by cxxbridge. No idea what this is used for.